packages/routing-sdk/addons/ton/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@pancakeswap/ton-addon",
```

Comment:
can rename to `@pancakeswap/routing-sdk-addon-ton` by convention

Reply:
modifed.